core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -462,4 +480,45 @@ public void unmount(AlluxioURI path, UnmountOptions options)
       mFileSystemContext.releaseMasterClient(masterClient);
     }
   }
+
+  /**
+   * Alluxio URIs in the Filesystem don't need a scheme or host. Warn the user and throw an
+   * exception if necessary.
+   */
+  protected static void UriCheck(AlluxioURI uri) {
```

Comment:
I'd make this private and test by calling the public methods. In general it's better to only test public methods so that internal refactoring doesn't break the tests, and it also covers more by ensuring that we're using `UriCheck` in the right place

Reply:
+1 for splitting each case to its own method. It won't add much repetition, and it's nice for each component of the test to be able to pass or fail individually. Shorter tests are easier to debug